Source/Fuse.Controls.Panels/LayoutControl.uno

```diff
@@ -193,5 +193,16 @@ namespace Fuse.Controls
 				return LayoutDependent.Maybe;
 			}
 		}
+
+		protected override bool FastTrackDrawWithOpacity(DrawContext dc)
+		{
+			if (HasChildren) return false;
+			
+			if (Background != null)
+				DrawBackground(dc, Opacity);
+			
+			// Asserting base class doesn't need to draw anything!
```

Comment:
Isn't it just a typo for "Assuming"?

But, wouldn't it be better to do something like:
```uno
protected override bool FastTrackDrawWithOpacity(DrawContext dc)
{
	if (HasChildren)
		return false;

	return base.FastTrackDrawWithOpacity(dc);
}
```

I mean, LayoutControl inherits from Control, which already does the above...